src/app/app-utils.ts

```diff
@@ -157,4 +157,22 @@ export class AppUtils {
     }
     return null;
   }
+
+  public static convertDomainToLabel(userRole: string): string {
```

Comment:
I have a test already for this in my pr https://github.com/hmcts/rpx-xui-webapp/pull/1308

Reply:
Yes, I actually copied your test. The reason is because Sonar was complaining that I had new code with no tests so have to put this in